### DISCUSSION TITLE
Tart version on Homebrew

### DISCUSSION THREAD

**Discussion by valmnt** [cirruslabs/tart#482] [General]
Hi,

I was wondering why Tart on Homebrew was still in 0.38 knowing that on Github the latest version is 1.4.

Thx.

**Comment by fkorotkov** [comment]
Tart formula that is getting installed via `brew install tart` is no the one officially supported by us. This formula builds Tart from sources which has two main problems:

1. Since the binary is not signed by `Cirrus Labs` developer certificate that we obtained from Apple, this binary will require authorization to access Keychain each time you update it.
2. In order to use Bridged network, one need special entitlements that can only be obtained after Apple's review and approval. We got went through the process so Tart can support bridged networking.

Please use our official tap where the updates are immediately available after release. Our CI process builds binaries, signs them and notarizes with Apple so you don't have issues mentioned above.

```bash
brew uninstall tart
brew install cirruslabs/cli/tart
```

**Comment by naikrovek** [comment]
The version of Tart in Homebrew is chosen (I assume) because it is the last open source version.  After 0.38, the version number was incremented to 1.0 and a new license was applied that is not an open source license.  I suspect, but don't know, that this is the reason the default Formula for Tart is still on 0.38.
